kinto/tests/test_config.py

```diff
@@ -0,0 +1,25 @@
+from ..config import render_template as origin
```

Comment:
You need to import with absolute locations. `from kinto.config import render_template`. You don't need to import it as origin.